nb/test/test.md

```diff
@@ -1,3 +1,94 @@
+![Image](test.md.joint.png)
```

Comment:
This looks cool! What is it?

Reply:
I've been making a stand for the steam deck, and wanted a joint that could fold into a triangle.

Although I realized a little while ago that I modeled it incorrectly -- instead of a joint with two arms, I should be modeling a half-hinge with an arm. :)